### PR TITLE
docs: updating readmes

### PR DESCRIPTION
**Pull request by emilyjablonski**
# Pull Request Template

## Issue

Addresses #1534

- [x] This change addresses the issue in full
- [ ] This change addresses only certain aspects of the issue
- [ ] This change is a dependency for another issue
- [ ] This change has a dependency from another issue

Updated all the READMEs in prep for Yazeed starting on Monday to ensure everything is in a good state. They were pretty outdated in some cases 😬

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,81 +10,86 @@ The frontend apps can easily be deployed to any Jamstack-friendly web host such
...
+
+### Pull Requests
+
+Pull requests are opened to the dev branch, not to master. When opening a pull request please fill out the entire pull request template which includes tagging the issue your PR is related to, a description of your PR, indicating the type of change, including details for the reviewer about how to test your PR, and a testing checklist. Additionally, officially link the issue in GitHub's right-hand panel.
+
+Every PR needs to manually update our changelog. Find the relevant section (General, Frontend, Backend, UI Components) and subsection (Added, Changed, Fixed) and add a short description of your change followed by a link to the PR and your name (- Description Here ([#1234](https://github.com/bloom-housing/bloom/pull/1234)) (Your Name)). If it is a breaking change, please include **Breaking Change** and some notes below it about how to migrate.
+
+It is not required, but adding the ready for review label onto your PR when it is ready will help surface it to our internal team, as will assigning specific reviewers.
```
Comment (seanmalbert) [inline README.md:93]: Should we make it required, in terms of process? And should it also say something about the WIP tag? I'd also like something for if they're the reviewer.

Something like:
If you are reviewing a PR, try not to leave only comments. If the PR requires further discussion or changes, mark it with `Request Changes`. If a PR looks good to you or there are changes request that won't require an additional review, please mark it with `Approved`. This helps other reviewers better understand the state of PRs at the list view.

File: backend/core/README.md [hunk 2]
```diff
@@ -8,22 +8,47 @@ OpenAPI (fka Swagger) documentation is automatically generated by the server at
...
+### Install Node
+
+Install Node.js 14.x
+
+`brew install node@14.`
+
 ### Installing Postgres
 
-You can install Postgres using Homebrew with the following command: `brew install postgresql`.
+You can install Postgres 12 using Homebrew with the following command: `brew install postgresql`.
```
Comment (seanmalbert) [inline backend/core/README.md:25]: I think I recall having to do another step like setting up a user, but we can update this if Yazeed runs into any issues.

File: sites/partners/README.md [hunk 3]
```diff
@@ -1,39 +1,31 @@
...
-  "type": "node",
-  "restart": true,
-  "sourceMaps": true,
-  "cwd": "${workspaceFolder}/sites/partners"
-},
-```
+To run the Cypress suite, with the application running, run `yarn test` from within `sites/partners` and when the test runner in a Chrome browser opens, click on whichever suite you want to run.
 
-## Running end-to-end tests locally
+## Environment Variables
```
Comment (seanmalbert) [inline sites/partners/README.md:19]: Should we have the same thing for the backend README?

**Review by seanmalbert — APPROVED** [review]
@emilyjablonski ,
These updates look great! Thank you! I have some comments/suggestions. Once you're satisfied with it, please merge.

**Comment by emilyjablonski** [comment]
Updates made, thanks!

**Pull request merged**